### ISSUE TITLE
How to properly uninstall the MultiplayerSleep-Datapack?

### ISSUE THREAD

**Issue by Anchor5178** [Plagiatus/datapacks#12]
Since installing the Datapack was just uploading the .zip into the datapacks folder, I removed it by deleting the .zip again but after restarting the server it prints the following:
`[Server thread/WARN]: Missing data pack file/multiplayer_sleepv2.1.0.zip`

**Comment by Plagiatus** [comment]
yeah, it should be just removing the datapack again that does the trick.
It might warn you because it still had it in its list of activated datapacks and couldn't find the files anymore. Now it should just be done from the list and you shouldn't get the warning next time you start the server.
